test/util/common.go

```diff
@@ -388,6 +371,40 @@ func SetupFiveValChain(t *testing.T) (TestInput, sdk.Context) {
 	return input, input.Context
 }
 
+func CreateValidator(
+	t *testing.T,
+	input TestInput,
+	accAddr sdk.AccAddress,
+	accPubKey ccrypto.PubKey,
+	accountNumber uint64,
+	valAddr sdk.ValAddress,
+	consPubKey ccrypto.PubKey,
+	stakingAmount cosmosmath.Int,
+	evmAddr gethcommon.Address,
+) {
+	// Initialize the account for the key
+	acc := input.AccountKeeper.NewAccount(
+		input.Context,
+		authtypes.NewBaseAccount(accAddr, accPubKey, accountNumber, 0),
+	)
+
+	// Set the balance for the account
+	require.NoError(t, input.BankKeeper.MintCoins(input.Context, types.ModuleName, InitCoins))
+	// nolint
```

Comment:
[not blocking] because I see this was moved but can the lint disable please document why the linter is turned off and scope down to just one linter? I.e.

```go
        // disable linter because it doesn't matter if send coins fails in this test
	//nolint:errcheck
	input.BankKeeper.SendCoinsFromModuleToAccount(input.Context, types.ModuleName, acc.GetAddress(), InitCoins)
```

or preferably don't disable the linter

```suggestion
	err := input.BankKeeper.SendCoinsFromModuleToAccount(input.Context, types.ModuleName, acc.GetAddress(), InitCoins)
	assert.NoError(t, err)
```